src/main/java/com/contrastsecurity/http/UrlBuilder.java

```diff
@@ -39,6 +39,10 @@ public String getApplicationsUrl(String organizationId) {
         return String.format("/ng/%s/applications?%s", organizationId, "base=false");
     }
 
+    public String getApplicationsNameUrl(String organizationId) {
+        return String.format("/ng/%s/applications/name?%s", organizationId, "base=false");
```

Comment:
We can clip base=false off of this as it looks like /name doesn't check for base. Don't forget to remove the `%s` off of the first argument.